lang/apple-gcc40/Portfile

```diff
@@ -11,12 +12,19 @@ long_description	Apple's version of the GNU compiler collection, \
 			version 4.0. Supports C and Objective-C only.
 
 homepage		http://opensource.apple.com/
-master_sites		http://opensource.apple.com/tarballs/gcc_40
-distname		gcc_40-${version}
+master_sites	https://github.com/apple-oss-distributions/gcc/archive/
+dist_subdir     apple-oss-distributions
+extract.rename  yes
+distname		gcc-${version}
 
-checksums           sha1    07574623e23103cb4024410ca48481fbb5da322e \
-                    rmd160  d7a5d15d312b9d7295564cf3862ec63296e769a3 \
-                    sha256  86b9c1d48c30042ba23d181a58ceeb3afb850b595206d976f78c985baea4b5ad
+distfiles       gcc-5493${extract.suffix}
+# Apple's github had missed 5494 build but it had only 5493,
+# here recovery minimal differences betwen them
+patchfiles      apple-gcc-5493-5494.diff
```

Comment:
If this one is missing upstream, it's probably fine to just use macports_distfiles for the master_sites.

Reply:
@jmroot but that patch is extremely small; I'd like to say almost nothing.

Reply:
yeah, the patch is only 51 lines, of which only 2 are functional (3 if you include the version number bump)